doc/classes/PhysicsServer3D.xml

```diff
@@ -1340,11 +1340,11 @@
 		<constant name="BODY_MODE_KINEMATIC" value="1" enum="BodyMode">
 			Constant for kinematic bodies. In this mode, a body can be only moved by user code and collides with other bodies along its path.
 		</constant>
-		<constant name="BODY_MODE_DYNAMIC" value="2" enum="BodyMode">
-			Constant for dynamic bodies. In this mode, a body can be pushed by other bodies and has forces applied.
+		<constant name="BODY_MODE_RIGID" value="2" enum="BodyMode">
```

Comment:
Nit: Would it be better to leave these as `BODY_MODE_DYNAMIC*`?  Would they apply to soft bodies or other body types?

Reply:
You need to understand how the bodies are organised internally to use the PhysicServer.

All bodies are RigidBodies with different modes. For example, a StaticBody is a RigidBody with infinite mass. As this can be difficult for a beginner to understand, there are several nodes.

 The only exception is the SoftBody which inherits from MeshInstance.

In one of the two proposals the user has made a small tree that may help to understand.

Reply:
Got it, thanks for clarifying!